<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saharwindow</class>
 <widget class="QMainWindow" name="saharwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>saharwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidgetS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>CRUD</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>351</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>AJOUTER</string>
      </property>
      <widget class="QPushButton" name="pushButtonajouterjou">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

padding: 1px;

border-style: solid;

border: 2px solid gray;

border-radius: 8px;

background-color: rgb(143, 143, 143);

border:2px solid #000;

}

QPushButton {

color: #fff !important;

text-transform: uppercase;

text-decoration: none;

background: #e12d4e;

padding: 20px;

border-radius: 5px;

display: inline-block;

border: none;

transition: all 0.4s ease 0s;

}

QPushButton:hover {

background: #434343;

letter-spacing: 1px;

-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

box-shadow: 5px 40px -10px rgba(0,0,0,0.57);

transition: all 0.4s ease 0s;

}</string>
       </property>
       <property name="text">
        <string>AJOUTER</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditja">
       <property name="geometry">
        <rect>
         <x>206</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="combopaysja">
       <property name="geometry">
        <rect>
         <x>207</x>
         <y>120</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nouvel élément</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="labelNomAJ">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM_PREN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_datenaisaj">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DATE_NAIS</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEditja">
       <property name="geometry">
        <rect>
         <x>206</x>
         <y>90</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomja">
       <property name="geometry">
        <rect>
         <x>206</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_paysaj">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>PAYS</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_specialiteaj">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SPECIALITE</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combospja">
       <property name="geometry">
        <rect>
         <x>207</x>
         <y>150</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Foot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Natation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tenis</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButtonajouterjou_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>190</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

padding: 1px;

border-style: solid;

border: 2px solid gray;

border-radius: 8px;

background-color: rgb(143, 143, 143);

border:2px solid #000;

}

QPushButton {

color: #fff !important;

text-transform: uppercase;

text-decoration: none;

background: #e12d4e;

padding: 20px;

border-radius: 5px;

display: inline-block;

border: none;

transition: all 0.4s ease 0s;

}

QPushButton:hover {

background: #434343;

letter-spacing: 1px;

-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

box-shadow: 5px 40px -10px rgba(0,0,0,0.57);

transition: all 0.4s ease 0s;

}</string>
       </property>
       <property name="text">
        <string>MODIFIER</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>341</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>SUPPRIMER</string>
      </property>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>130</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

padding: 1px;

border-style: solid;

border: 2px solid gray;

border-radius: 8px;

background-color: rgb(143, 143, 143);

border:2px solid #000;

}

QPushButton {

color: #fff !important;

text-transform: uppercase;

text-decoration: none;

background: #e12d4e;

padding: 20px;

border-radius: 5px;

display: inline-block;

border: none;

transition: all 0.4s ease 0s;

}

QPushButton:hover {

background: #434343;

letter-spacing: 1px;

-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

box-shadow: 5px 40px -10px rgba(0,0,0,0.57);

transition: all 0.4s ease 0s;

}</string>
       </property>
       <property name="text">
        <string>SUPPRIMER</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combosupj">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableViewja">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>711</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Metiers</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>490</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

padding: 1px;

border-style: solid;

border: 2px solid gray;

border-radius: 8px;

background-color: rgb(143, 143, 143);

border:2px solid #000;

}

QPushButton {

color: #fff !important;

text-transform: uppercase;

text-decoration: none;

background: #e12d4e;

padding: 20px;

border-radius: 5px;

display: inline-block;

border: none;

transition: all 0.4s ease 0s;

}

QPushButton:hover {

background: #434343;

letter-spacing: 1px;

-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

box-shadow: 5px 40px -10px rgba(0,0,0,0.57);

transition: all 0.4s ease 0s;

}</string>
      </property>
      <property name="text">
       <string>fichehistorique</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>504</x>
        <y>410</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

padding: 1px;

border-style: solid;

border: 2px solid gray;

border-radius: 8px;

background-color: rgb(143, 143, 143);

border:2px solid #000;

}

QPushButton {

color: #fff !important;

text-transform: uppercase;

text-decoration: none;

background: #e12d4e;

padding: 20px;

border-radius: 5px;

display: inline-block;

border: none;

transition: all 0.4s ease 0s;

}

QPushButton:hover {

background: #434343;

letter-spacing: 1px;

-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

box-shadow: 5px 40px -10px rgba(0,0,0,0.57);

transition: all 0.4s ease 0s;

}</string>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>410</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

padding: 1px;

border-style: solid;

border: 2px solid gray;

border-radius: 8px;

background-color: rgb(143, 143, 143);

border:2px solid #000;

}

QPushButton {

color: #fff !important;

text-transform: uppercase;

text-decoration: none;

background: #e12d4e;

padding: 20px;

border-radius: 5px;

display: inline-block;

border: none;

transition: all 0.4s ease 0s;

}

QPushButton:hover {

background: #434343;

letter-spacing: 1px;

-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);

box-shadow: 5px 40px -10px rgba(0,0,0,0.57);

transition: all 0.4s ease 0s;

}</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>449</x>
        <y>320</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RECHERCHER</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>539</x>
        <y>320</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>320</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>350</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>TRI</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_7">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>350</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewja_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>791</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewja_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>341</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
